File: css/lucky_number.cpp
#include <iostream>
using namespace std;
int main() {
    int n, sum = 0;
    cin >> n;
    while(n) { sum += n % 10; n /= 10; }
    cout << ((sum % 9 == 0) ? "🎉 Lucky!" : "❌ Not lucky.") << endl;
    return 0;
}

File: erlang/lucky_number.java
import java.util.Scanner;
public class LuckyNumber {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt(), sum = 0;
        while(n != 0) { sum += n % 10; n /= 10; }
        System.out.println((sum % 9 == 0) ? "🎉 Lucky!" : "❌ Not lucky.");
    }
}
